### ISSUE TITLE
Adding material check fails with 400 (Bad Request) against Nexus NuGet repository

### ISSUE THREAD

**Issue by clacivita** [ThoughtWorksInc/go-nuget-poller-plugin#22]
i'm using the open-source version of the Nexus repository manager. I can successfully configure the repository in Go. However, when adding a new package I'm seeing the following error when I click the "Check Package" button;

Package check Failed. Reason(s): RuntimeException (HTTP 400, Bad Request) while getting package feed for : http://username:password@10.10.10.10:8081/nexus/service/local/nuget/Releases/GetUpdates()?packageIds='MyPackageID'&versions='0.0.1'&includePrerelease=true&includeAllVersions=true&$orderby=Version%20desc&$top=1

The actual NuGet repository works fine through Go. NuGet at the command line works fine as well.

**Comment by clacivita** [comment]
Of course, the minute after I send this I get an answer. Nexus doesn't support the GetUpdates() call. https://issues.sonatype.org/browse/NEXUS-6372. 

I don't suppose there is a way to make the plugin use the older ODATA query instead?

**Comment by ChristopherElliott** [comment]
Interesting. I saw the same issue with Microsoft's Visual Studio Team Services "Package Management". I wonder if there is a common reason why GetUpdates() is no longer being implemented even when a v2 endpoint is supposedly supported.
